src/Enum.php

```diff
@@ -81,7 +81,7 @@ public function __toString(): string
      * @throws LogicException Enums are not cloneable
      *                        because instances are implemented as singletons
      */
-    final private function __clone()
+    private function __clone()
```

Comment:
Should it be `final protected` then, to prevent overwriting the method?

Reply:
Cool. `final protected` is more better. Let me update.